cluster/ubuntu/util.sh

```diff
@@ -457,3 +461,24 @@ function kube-push {
 function prepare-e2e() {
   echo "Ubuntu doesn't need special preparations for e2e tests" 1>&2
 }
+
+# Set proxy environment variables
+function setProxy() {
+  source "kube/${KUBE_CONFIG_FILE-"config-default.sh"}"
```

Comment:
You shouldn't need to source it here. All functions in this file depends on variables in config-default.sh.